spec/unit/s3/responses/empty_spec.rb

```diff
@@ -8,7 +8,7 @@
   end
 
   it "has an empty body" do
-    expect(subject.body).to be_empty
+    expect(subject.body).to eq([''])
```

Comment:
`Rack` [body response](http://rack.github.io/) is supposed to be returned as an enumerable.